### PR TITLE
Refunds amounting to 0 ignored

### PR DESCRIPTION
**Pull request by jnmrclmbsjse**
This PR includes:
1. Refunds with 0 amount will be marked as synchronized
2. Unit test

### CODE REVIEW

**Comment by jnmrclmbsjse** [comment]
Task linked: [CU-8692yv4dp Refund amount needs to be negative](https://app.clickup.com/t/8692yv4dp)

File: src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php [hunk 1]
```diff
@@ -861,50 +861,58 @@ protected function doProcessRefundsWithoutIds($unsyncedRefundsWithoutId, $woocom
...
+                $storekeeperRefundId = PaymentGateway::createRefundAsPayment($refundId, $refundAmount);
+                $this->debug('Storekeeper refund was created', [
+                    'order_id' => $storekeeperId,
+                    'payment_id' => $storekeeperRefundId,
+                ]);
+                PaymentGateway::updateRefund($id, $storekeeperRefundId, $refundAmount);
+                $this->syncPaymentToBackend($storekeeperRefundId, $storekeeperId);
             }
-
-            PaymentGateway::markRefundAsSynced($woocommerceOrderId, $storekeeperRefundId, $refundId);
```
Comment (ikulis) [inline src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php:899]: why was this line removed

Comment (jnmrclmbsjse) [reply src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php:899]: Both if and else calls `PaymentGateway::markRefundAsSynced($woocommerceOrderId, $storekeeperRefundId, $refundId);` and PHPStan flagged it so I moved it outside
